sync/class.jetpack-sync-module-callables.php

```diff
@@ -145,7 +145,7 @@ public function maybe_sync_callables() {
 			return;
 		}
 
-		$callable_checksums = (array) get_option( self::CALLABLES_CHECKSUM_OPTION_NAME, array() );
+		$callable_checksums = (array) Jetpack_Options::get_raw_option( self::CALLABLES_CHECKSUM_OPTION_NAME, array() );
```

Comment:
From memory, I noticed that the callables weren't syncing as often as I expected them to. Once I made this change, it cleared up.